apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -264,8 +264,8 @@ <h3 class="ledger-main-head">
                             <div class="px-3 w-100">
                                 <table class="ledger-head transaction-mapped mt-0" width="100%" colspan="100%">
                                     <tr>
-                                        <th colspan="6" class="pl-15 pr-2 font-14">Debit <span class="pull-right text-gray">Dr</span></th>
-                                        <th colspan="6" class="pr-15 pl-1 font-14">Credit <span class="pull-right text-gray">Cr</span></th>
+                                        <th colspan="6" class="pd-l15 pr-2 font-14">Debit <span class="pull-right text-gray">Dr</span></th>
+                                        <th colspan="6" class="pd-r15 pl-1 font-14">Credit <span class="pull-right text-gray">Cr</span></th>
```

Comment:
check for this pl-1 also, if it is working fine

Reply:
Updated